apps/Gruntfile.js

```diff
@@ -8,6 +8,7 @@ var _ = require('lodash');
 var logBuildTimes = require('./script/log-build-times');
 var webpackConfig = require('./webpack');
 var envConstants = require('./envConstants');
+var ChunkManifestPlugin = require('chunk-manifest-webpack-plugin');
```

Comment:
I'm not sure whether I need this yet. Supposedly it helps with client side caching but I'm not sure exactly how it works yet. I'll remove it before merging.